test/tests/functional/pbs_que_resc_usage.py

```diff
@@ -71,31 +68,30 @@ def test_resc_assigned_set_unset(self):
         """
 
         # set a resource and unset it without using
-        a = {'resources_available.ncpus': 4}
-        self.server.manager(MGR_CMD_SET, QUEUE, a, id='workq')
+        self.server.manager(MGR_CMD_SET, QUEUE, {
+                            'resources_available.ncpus': 4}, id='workq')
         q_status = self.server.status(QUEUE, id='workq')
         self.assertEqual(
             int(q_status[0]['resources_available.ncpus']), 4, self.err_msg)
         self.assertNotIn('resources_assigned.ncpus',
-                         q_status[0].keys(), self.err_msg)
+                         q_status[0], self.err_msg)
         self.server.manager(MGR_CMD_UNSET, QUEUE,
                             'resources_available.ncpus', id='workq')
         q_status = self.server.status(QUEUE, id='workq')
         self.assertNotIn('resources_available.ncpus',
-                         q_status[0].keys(), self.err_msg)
+                         q_status[0], self.err_msg)
 
         # set the resource and unset it after using
-        a = {'resources_available.ncpus': 8}
-        self.server.manager(MGR_CMD_SET, QUEUE, a, id='workq')
+        self.server.manager(MGR_CMD_SET, QUEUE, {
+                            'resources_available.ncpus': 8}, id='workq')
```

Comment:
While both the old and new code are pep8 compliant, I find the old code more readable (here and above)